subjects/java/checkpoints/longest-common-prefix/README.md

```diff
@@ -0,0 +1,50 @@
+## Longest Common Prefix
+
+### Instructions
+
+Write a function to find the longest common prefix string amongst an array of strings. If there is no common prefix, return an empty string `""`.
```

Comment:
Nothing major, but I would specify if the prefix is supposed to be case-sensitive or not. In other exercises the comparison was not case-insensitive while here it is case-sensitive (I only found out looking at the tests)

Reply:
yes, you are right i forget to add that.
its fixed now thanks